frontend/android/app/src/main/AndroidManifest.xml

```diff
@@ -1,49 +1,82 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
           xmlns:tools="http://schemas.android.com/tools"
-    package="app.ehrenamtskarte">
+          package="app.ehrenamtskarte">
     <!-- io.flutter.app.FlutterApplication is an android.app.Application that
          calls FlutterMain.startInitialization(this); in its onCreate method.
          In most cases you can leave this as-is, but you if you want to provide
          additional functionality it is fine to subclass or reimplement
          FlutterApplication and put your custom class here. -->
     <application
-        android:label="@string/BUILD_CONFIG_APP_NAME"
-        android:icon="@mipmap/${icon}"
-        android:roundIcon="@mipmap/${icon}_round">
+            android:label="@string/BUILD_CONFIG_APP_NAME"
+            android:icon="@mipmap/${icon}"
+            android:roundIcon="@mipmap/${icon}_round">
         <activity android:exported="true"
-            android:name=".MainActivity"
-            android:launchMode="singleTop"
-            android:theme="@style/LaunchTheme"
-            android:configChanges="orientation|keyboardHidden|keyboard|screenSize|smallestScreenSize|locale|layoutDirection|fontScale|screenLayout|density|uiMode"
-            android:hardwareAccelerated="true"
-            android:windowSoftInputMode="adjustResize">
+                  android:name=".MainActivity"
+                  android:theme="@style/LaunchTheme"
+                  android:configChanges="orientation|keyboardHidden|keyboard|screenSize|smallestScreenSize|locale|layoutDirection|fontScale|screenLayout|density|uiMode"
+                  android:hardwareAccelerated="true"
+                  android:launchMode="singleTask"
```

Comment:
Can you please explain why you decided to go to "singleTask" mode. This is a specialized launch and not recommended for general use.

Reply:
reverted to `singleTop` my mistake